component-samples/demo/owner/README.md

```diff
@@ -43,7 +43,7 @@ All the runtime configurations for the services are specified in four files: `se
 
 `service.yml` file is structured into multiple sections:
 
-- `hibernate-properties:` - This section contains hibernate related runtime properties including the DB URL, dialect and others.
+- `hibernate-properties:` - This section contains *Hibernate related runtime properties including the DB URL, dialect and others.
```

Comment:
Should it be *Hibernate or Hibernate*

Reply:
*Hibernate -- For external dependencies, we add asterisk `*`